lib/helpers.js

```diff
@@ -47,6 +57,72 @@ function fetchAndCache(request, options) {
   });
 }
 
+var staticIdb;
+function idbPromise() {
+  if (staticIdb) {
+    return Promise.resolve(staticIdb);
+  }
+
+  return simpleDB.open('sw-toolbox-lru-order').then(function(idb) {
+    staticIdb = idb;
+    return staticIdb;
+  });
+}
```

Comment:
Since we only ever return a promise, we might as well store the promise rather than the value.

Alternative:
```javascript
function idbPromise() {
  if (staticIdb) {
    return staticIdb;
  }

  return staticIdb = simpleDB.open('sw-toolbox-lru-order');
}
```